edxval/settings/test.py

```diff
@@ -17,3 +17,6 @@
 TEST_ONLY_URLS = True
 PROVIDER_BASE_URL = 'http://127.0.0.1:8000'
 PROVIDER_STATES_URL = '{}/edxval/pact/provider_states/'.format(PROVIDER_BASE_URL)
+
+MIDDLEWARE = list(MIDDLEWARE) + ['edxval.pacts.middleware.AuthenticationMiddleware']
+MIDDLEWARE = tuple(MIDDLEWARE)
```

Comment:
Can be improved to this: 
`MIDDLEWARE = MIDDLEWARE + ('edxval.pacts.middleware.AuthenticationMiddleware',)`